guacamole-common-js/src/main/webapp/modules/BlobWriter.js

```diff
@@ -110,6 +110,10 @@ Guacamole.BlobWriter = function BlobWriter(stream) {
         var offset = 0;
         var reader = new FileReader();
 
+        var sleep = function sleep(ms) {
+            return new Promise(resolve => setTimeout(resolve, ms));
+        }
```

Comment:
Beware you're missing a semicolon here. Not an issue if this function is ultimately deleted (see below).